### PR TITLE
Added method to make use of `Mage_Core_Model_Security_HtmlEscapedString` easier

### PR DESCRIPTION
**Pull request by sreichel**
- added new methods to not have initialize class manually
- added missing docblocs + copyright header
- updated/unified some docblock

Better name for that method?

### CODE REVIEW

**Review by Flyingmana — APPROVED** [review]

**Comment by sreichel** [comment]
@Flyingmana thanks for review, but i dont like the method-name ... any better suggestions?

- escapeHtmlAsSecureObject
- escapeHtmlAndKeepOriginalValue
- escapeHtmlWithKeepOriginalValue
- ...

File: app/code/core/Mage/Core/Model/Security/HtmlEscapedString.php [hunk 1]
```diff
@@ -3,12 +3,35 @@
...
+ * It is also available at https://opensource.org/license/osl-3-0-php
+ *
+ * @category   Mage
+ * @package    Mage_Core
+ * @copyright  Copyright (c) 2024 The OpenMage Contributors (https://www.openmage.org)
+ * @license    https://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+/**
+ * Wrapper to escape value und keep the original value
```
Comment (kiatng) [inline app/code/core/Mage/Core/Model/Security/HtmlEscapedString.php:19]:
```suggestion
 * Wrapper to escape a string value with a method to get the original string value
```

**Comment by kiatng** [comment]
> @Flyingmana thanks for review, but i dont like the method-name ... any better suggestions?
> 
> * escapeHtmlAsSecureObject
> * escapeHtmlToSecuredObject
> * escapeHtmlAndKeepOriginalValue
> * escapeHtmlWithKeepOriginalValue
> * ...

Name it as it is: `getCoreModelSecurityHtmlEscapedString()`, `getCoreModelsSecurityHtmlEscapedString()`

**Comment by sreichel** [comment]
I do not like such long names, but finally you are right. "Name it as it is" :+1:

**Review by kiatng — DISMISSED** [review]

**Review by ADDISON74 — DISMISSED** [review]

**Review by kiatng — APPROVED** [review]

**Review by ADDISON74 — APPROVED** [review]

**Pull request merged**